### PR TITLE
Update theming docs

### PR DESCRIPTION
**Pull request by borisyordanov**
I tweaked a bit the theming documentation, in regards to the breakpoints. Please verify that what i have written is correct and let me know if i can improve this PR somehow.

### CODE REVIEW

File: src/theming.md [hunk 1]
```diff
@@ -16,12 +16,15 @@ export default props =>
...
 
+If you do not apply your own theme, or you don't include all the keys the theme object expects, Rebass will revert to the default values, which are inherited from [styled-system][].
+
 An example theme could look like the following:
 
 ```js
 // example theme.js
 
 export default {
+  breakpoints: [40, 52, 64],
```
Comment (jxnblk) [inline src/theming.md]: These should be strings with CSS units (e.g. `'40em'`)

File: src/theming.md [hunk 2]
```diff
@@ -64,6 +67,8 @@ Key | Type | Description
 `radii` | Array or Object | Values for `borderRadius` props
 `opacity` | Array or Object | Values for `opacity` props
 
+By default arrays of numbers are interpreted as `em`, other unit of measurements can be specified by using an array of strings instead, e.x. `['768px', '992px', '1200px']`.
```
Comment (jxnblk) [inline src/theming.md]: This changed in a previous version of styled-system – all numbers are converted to `px` units if they're not strings, but I generally recommend using strings, especially for breakpoints. See https://github.com/jxnblk/styled-system/blob/master/src/index.js#L61

Comment (borisyordanov) [reply src/theming.md]: I made some updates after i saw your comments. Please review again.

I'm curious - what made you decide that numbers should be converted to px (and not another unit)?

Comment (jxnblk) [reply src/theming.md]: Awesome, thanks! The pixel thing come from how React interprets numbers in `style` objects

**Comment by jxnblk** [comment]
Thanks for this!

**Pull request merged**